test-main.js

```diff
@@ -0,0 +1,24 @@
+var allTestFiles = []
+var TEST_REGEXP = /(spec|test)\.js$/i
+
+// Get a list of all the test files to include
+Object.keys(window.__karma__.files).forEach(function (file) {
```

Comment:
This is needed by Karma. Otherwise I seem to experience a delay in the starting of tests.